setup.py

```diff
@@ -59,7 +59,7 @@ def run(self):
     "Django~=2.2.18",
     "galaxy-importer==0.2.15",
     "pulpcore<3.11,>=3.10",
-    "pulp-ansible==0.7.0",
+    "pulp-ansible",
```

Comment:
are we going to leave pulp_ansible unpinned?

Reply:
I feel we should keep pulp_ansible pinned, unless we found a reason we need to change